libretroshare/src/pqi/p3peermgr.cc

```diff
@@ -1119,13 +1130,9 @@ bool p3PeerMgrIMPL::addSslOnlyFriend( const RsPeerId& sslId, const RsPgpId& pgp_
 	if(dt.hiddenType)        pstate.hiddenType = dt.hiddenType;
 	if(!dt.location.empty()) pstate.location = dt.location;
 
-    pstate.skip_pgp_signature_validation = true;
```

Comment:
This is the place where it should be: after all checks.

Reply:
see my explaination upster ;)

Reply:
I'm thinking in terms of code security. This flag is a bomb: if some bug or hack flips it, the software's security is down. That's why it's preferable to set it after all tests are done, so that any further change in the code is less likely to unintentionally cause such a problem. Maybe you can add that comment before setting the flag.

Reply:
Ok I agree in commenting this on the code